M2/Macaulay2/packages/CellularResolutions.m2

```diff
@@ -8,7 +8,7 @@ newPackage(
         {Name => "Jay Yang", Email => "jayy@wustl.edu"},
         {Name => "Aleksandra Sobieska", Email => "asobieska@math.wisc.edu"}
         },
-    Headline => "A package for cellular resolutions of monomial ideals",
+    Headline => "cellular resolutions of monomial ideals",
```

Comment:
I'm trying to make all package headlines uniform, which includes removing redundant clauses. @jkyang92 is this change okay with you?